scripts/entity/tactical/enemies/rf_bandit_baron.nut

```diff
@@ -156,23 +158,8 @@ this.rf_bandit_baron <- ::inherit("scripts/entity/tactical/human", {
 		local r = ::Math.rand(1, 100);
 		if (r <= 25)
 		{
-			local weapon = ::MSU.Class.WeightedContainer([
-				[1, "scripts/items/weapons/named/named_axe"],
-				[1, "scripts/items/weapons/named/named_cleaver"],
-				[1, "scripts/items/weapons/named/named_spear"],
-				[1, "scripts/items/weapons/named/named_sword"],
-				[1, "scripts/items/weapons/named/named_hammer"],
-				[1, "scripts/items/weapons/named/named_mace"],
-
-				[1, "scripts/items/weapons/named/named_rf_longsword"],
-				[1, "scripts/items/weapons/named/named_rf_kriegsmesser"],
-				[1, "scripts/items/weapons/named/named_rf_swordstaff"],
-				[1, "scripts/items/weapons/named/named_two_handed_flail"],
-				[1, "scripts/items/weapons/named/named_two_handed_mace"],
-				[1, "scripts/items/weapons/named/named_greatsword"]
-			]).roll();
-
-			this.m.Items.equip(::new(weapon));
+			local weapon = ::Reforged.ItemTable.NorthernWeaponFrontlineMaxTier.roll({})
```

Comment:
No need to pass `{}` to roll. You can simply do `roll()` if you don't want to add any options.